### PR TITLE
A: (nsfw) https://pornstreams.co/jamesdeen-21-07-19-karma-rx-please-g…

### PR DESCRIPTION
**Pull request by Yuki2718**
…ive-me-permission-to-cum-xxx-1080p-mp4-wrb/ (related: https://github.com/uBlockOrigin/uAssets/issues/10957)
`https://pornstreams.co/jamesdeen-21-07-19-karma-rx-please-give-me-permission-to-cum-xxx-1080p-mp4-wrb/`

`||mmngqvbfuzenlk.com^` is popAds from `dood.so`.

### CODE REVIEW

File: easylist/easylist_adservers.txt [hunk 1]
```diff
@@ -8526,6 +8526,7 @@
 ||mmgads.com^
 ||mmismm.com^
 ||mmissedecon.info^
+||mmngqvbfuzenlk.com^
```
Comment (spirillen) [inline easylist/easylist_adservers.txt:8529]: Not seen

File: easylist/easylist_adservers.txt [hunk 2]
```diff
@@ -10341,6 +10342,7 @@
 ||poresresigntun.com^
 ||pornlemon.com^
 ||pornomixfree.com^
+||pornyhd.com^
```
Comment (spirillen) [inline easylist/easylist_adservers.txt:10345]: :heavy_check_mark:

File: easylist/easylist_adservers.txt [hunk 3]
```diff
@@ -10408,6 +10410,7 @@
 ||ppscdnqkgj.com^
 ||ppxbsbluutiwc.xyz^
 ||ppxldikxksuo.com^
+||pqorwmmlkf.com^
```
Comment (spirillen) [inline easylist/easylist_adservers.txt:10413]: Not seen

File: easylist/easylist_adservers.txt [hunk 4]
```diff
@@ -11684,6 +11687,7 @@
 ||runwaff.com^
 ||rusenov.com^
 ||rushcostscomparison.com^
+||rushonuco.com^
```
Comment (spirillen) [inline easylist/easylist_adservers.txt:11690]: Not seen

File: easylist/easylist_adservers.txt (hunk 3 above)
Comment (Yuki2718) [reply easylist/easylist_adservers.txt:10413]: Yep, I don't see it anymore but this was already added to EL.

**Pull request closed without merging**